detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/optional/MandatoryBracesForLoopsSpec.kt

```diff
@@ -0,0 +1,62 @@
+package io.gitlab.arturbosch.detekt.rules.style.optional
+
+import io.gitlab.arturbosch.detekt.test.lint
+import org.assertj.core.api.Assertions.assertThat
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class MandatoryBracesForLoopsSpec : Spek({
+    val subject by memoized { MandatoryBracesForLoops() }
+
+    describe("reports multi-line for loops should have braces") {
+
+        it("does not report with braces") {
+            val code = """
+            fun test() {
+                for (i in 0..10) {
+                    println(i)
+                }
+            }
+            """
+
+            assertThat(subject.lint(code)).isEmpty()
```

Comment:
Please use the newer API. This allows to verify the correctness of given code snippets with the help of the Kotlin compiler.

```suggestion
            assertThat(subject.compileAndLint(code)).isEmpty()
```